templates/registration/profile.html

```diff
@@ -6,34 +6,33 @@
 {% endblock %}
 
 {% block content %}
-    <h1>Profile</h1>
-    <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_username %}!</h2>
+    <h2 class="profile-header">{% firstof user.get_full_name user.get_username %}'s Profile</h2>
     {% if user.userprofile.invest_anonymously %}<p><small>(displays publicly as Anonymous)</small></p>{% endif %}
 
     {% if artists %}
         {% if albums or updates %}
             <ul class="tabs" data-tabs id="profile-tabs">
-                <li class="tabs-title is-active"><a href="#portfolio">My Artists</a></li>
-                {% if albums %}
-                    <li class="tabs-title"><a href="#music">My Music</a></li>
-                {% endif %}
-                {% if updates %}
-                    <li class="tabs-title"><a href="#updates">Artist Updates</a></li>
-                {% endif %}
+                <li class="tabs-title is-active"><a href="#updates"><i class="fa fa-newspaper-o fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Activity</span></a></li>
+                <li class="tabs-title"><a href="#profile-stats"><i class="fa fa-bar-chart fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Earnings</span></a></li>
+                <li class="tabs-title"><a href="#portfolio"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Artists</span></a></li>
+                <li class="tabs-title"><a href="#music"><i class="fa fa-music fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Music</span></a></li>
             </ul>
 
             <div class="tabs-content" data-tabs-content="profile-tabs">
-                <div class="tabs-panel is-active" id="portfolio">
+                <div class="tabs-panel" id="portfolio">
                     {% include "registration/includes/profile_portfolio.html" %}
                 </div>
+                <div class="tabs-panel" id="profile-stats">
+                    {% include "registration/includes/profile_stats.html" %}
+                </div>
                 {% if albums %}
                     <div class="tabs-panel" id="music">
                         {% include "registration/includes/profile_music.html" %}
                     </div>
                 {% endif %}
                 {% if updates %}
-                    <div class="tabs-panel" id="updates">
-                        <h2>ARTIST UPDATES</h2>
+                    <div class="tabs-panel is-active" id="updates">
```

Comment:
Yes, you could do this. If you want to do that, then we should remove the `{% if %}` block around the updates panel.

Also, IMO the panels should be in the same order as the tabs. Otherwise, it's a bit confusing.